spec/spec.md

```diff
@@ -0,0 +1,180 @@
+followed by a `License-Text` tag, followed by the license text. Example:
+
+```
+Valid-License-Identifier: GPL-3.0-only
+Valid-License-Identifier: GPL-3.0-or-later
+License-Text:
+
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
+
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
+ Everyone is permitted to copy and distribute verbatim copies
+ of this license document, but changing it is not allowed.
+
+[...]
+```
+
+A project may contain a single license file outside of the `LICENSES/` directory
+in the file `COPYING`, `COPYRIGHT`, `LICENSE` or `LICENCE` in the root of the
+project.
+
+If a license does not exist in the SPDX License List, its SPDX identifier is
+`LicenseRef-[idstring]` as defined by the SPDX Specification.
+
+A project must not include license files for licenses under which none of the
+files in the project are licensed.
+
+Everything that applies to licenses in this section also applies to license
+exceptions, with the following exceptions:
+
+- Exception files must be placed in the `LICENSES/` directory in the root of the
+  project.
+
+- The tags `Valid-Exception-Identifier` and `Exception-Text` must be used
+  instead of `Valid-License-Identifier` and `License-Text`.
+
+<!-- Can we use LicenseRef for exceptions? -->
+
+<!-- TODO: Change "copyright and licensing information" to something shorter?
+Shorten it to CALI? -->
+
+## Copyright and licensing information
+
+Each file in the project must have copyright and licensing information
+associated with it, except the following files:
+
+- The license files.
+
+- The files belonging to the project's version control system (example:
+  `.git/`).
+
+- The files ignored by the version control system (example: `.gitignore`).
+
+- The files in the `.reuse/` directory in the root of the project.
+
+<!-- TODO: The files inside of a git submodule -->
+
+There are various ways to associate copyright and licensing information with a
+file.
+
+### Comment headers
+
+Each plain text file that can contain comments should contain comments at the
+top of the file (comment header) that declare that file's copyright and
+licensing information.
+
+If a file is not a plain text file or does not permit the inclusion of
+comments, the comment header that declares the file's copyright and licensing
+information should be in an adjacent file of the same name with the
+additional extension `.license` (example: `cat.jpg.license` if the original
+file is `cat.jpg`).
+
+The comment header must contain one or more `SPDX-Copyright` tags, and one or
+more `SPDX-License-Information` tags. A tag is followed by a colon, followed by
+a text value, and terminated by a newline.
+
+The `SPDX-Copyright` tag must be followed by a copyright notice as described
+in <!-- FIXME: Refer to an internal section -->.
+
+Instead of the `SPDX-Copyright` tag, the symbols `Copyright`, `Copyright (c)`,
+`Copyright (C)`, `Copyright ©` or `©` may be used.
```

Comment:
```suggestion
```

Reply:
Closing this in favour of a different convo